core/ssh_keys.go

```diff
@@ -0,0 +1,106 @@
+package core
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/krystal/go-katapult"
+)
+
+type AuthSSHKey struct {
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitempty"`
+	Fingerprint string `json:"fingerprint,omitempty"`
+}
+
+type SSHKeysClient struct {
+	client   RequestMaker
+	basePath *url.URL
+}
+
+func NewSSHKeysClient(rm RequestMaker) *SSHKeysClient {
+	return &SSHKeysClient{
+		client:   rm,
+		basePath: &url.URL{Path: "/core/v1/"},
+	}
+}
+
+type sshKeysResponseBody struct {
+	SSHKey  *AuthSSHKey   `json:"ssh_key,omitempty"`
+	SSHKeys []*AuthSSHKey `json:"ssh_keys,omitempty"`
+}
+
+func (s *SSHKeysClient) List(
+	ctx context.Context,
+	ref OrganizationRef,
+) ([]*AuthSSHKey, *katapult.Response, error) {
```

Comment:
The SSH Keys [endpoint](https://developers.katapult.io/docs/api/latest/core/routes/organizations/:organization/ssh_keys/get/) supports pagination. The `List()` method on `LoadBalancerRulesClient` should be a good reference for how we normally handle pagination :)